Doc/c-api/object.rst

```diff
@@ -6,6 +6,33 @@ Object Protocol
 ===============
 
 
+.. c:function:: PyObject* Py_GetConstant(unsigned int constant_id)
+
+   Get a :term:`strong reference` to a constant.
+
+   Return ``NULL`` if *constant_id* is invalid.
+
+   Constant identifiers:
+
+   .. c:macro: Py_NONE_IDX
+   .. c:macro: Py_FALSE_IDX
+   .. c:macro: Py_TRUE_IDX
+   .. c:macro: Py_ELLIPSIS_IDX
+   .. c:macro: Py_NOT_IMPLEMENTED_IDX
+   .. c:macro: Py_ZERO_IDX
+   .. c:macro: Py_ONE_IDX
+   .. c:macro: Py_EMPTY_STR_IDX
+   .. c:macro: Py_EMPTY_BYTES_IDX
+   .. c:macro: Py_EMPTY_TUPLE_IDX
+
+   .. versionadded:: 3.13
+
+
+.. c:function:: PyObject* Py_GetConstantBorrowed(unsigned int constant_id)
+
+   Similar to :c:func:`Py_GetConstant`, but return a :term:`borrowed
+   reference`.
```

Comment:
I think that for functions that return borrowed API, we should always explicitly document when the reference stays valid. Even if it's “forever”.
```suggestion
   Similar to :c:func:`Py_GetConstant`, but return a :term:`borrowed
   reference`.
   The reference is borrowed from the interpreter, and will be valid
   until interpreter finalization.
```

Reply:
I applied your change, thanks.